### DISCUSSION TITLE
How do i check if mediamtx is performing transcoding or not?

### DISCUSSION THREAD

**Discussion by AbhishekK1743** [bluenviron/mediamtx#3707] [Questions]
### Question

Thanks a lot for this amazing application. I am consuming stream from multiple rtsp url and publishing it to browser using webrtc in grid format. For 16 stream of 1280x720 resolution it is consuming ~100% cpu( one full core ). why does it consuming so much cpu? how do i check if mediamtx is doing transcoding ? if yes how do I stop transcoding? 

mediamtx version : mediamtx_v1.9.0_linux_arm64v8

below is the details of the machine where mediamtx is running : 
Architecture:        aarch64
Byte Order:          Little Endian
CPU(s):              4
On-line CPU(s) list: 0-3
Thread(s) per core:  1
Core(s) per socket:  4
Socket(s):           1
Vendor ID:           ARM
Model:               4
Model name:          Cortex-A53
Stepping:            r0p4
CPU max MHz:         1152.0000
CPU min MHz:         480.0000
BogoMIPS:            48.00

thank you for your time.

**Comment by aler9** [comment]
Hello, MediaMTX doesn't perform any transcoding. Nothing else can be asserted from the data you provided.
